src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -562,12 +562,15 @@ private static bool CanBindToTheseConstructorParameters(ParameterInfo[] construc
         // in their config class, then it is cloned to a new dictionary, the same way as other collections.
```

Comment:
(minor, on line 557 above)

> // Binds and potentially overwrites a concrete dictionary.

could be changed to:

// Binds and potentially overwrites a dictionary object.

Or something similar. With the new code, `a concrete dictionary` is no longer valid.